tests/admin_docs/tests.py

```diff
@@ -390,6 +390,8 @@ def test_model_docstring_renders_correctly(self):
             'reference external" href="/admindocs/models/myapp.company/">'
             'myapp.Company</a> where the person works.</dd></dl>'
         )
+        descriptionheading = '<h3>Description</h3>'
```

Comment:
I think I would put the test in `test_description_output` instead -- did you consider that?